src/test/java/teammates/test/cases/browsertests/InstructorSearchPageUiTest.java

```diff
@@ -109,7 +111,33 @@ private void testSearch() throws Exception {
         searchPage.verifyHtmlMainContent("/instructorSearchPageSearchStudentsForStudent2.html");
     }
 
+    private void testDeleteAction() throws Exception {
+
+        ______TS("action: delete");
+
+        searchPage.clearSearchBox();
+        String searchContent = "\"student2 2 In Course1\"";
+        searchPage.inputSearchContent(searchContent);
+        searchPage.clickSearchButton();
+        searchPage.verifyHtmlMainContent("/instructorSearchPageSearchStudentsForStudent2WithExactString.html");
```

Comment:
Was this repeat of the `search exact string for students` test case intentional?

Reply:
Yes, to double check that the search page has the correct content before testing the confirmation modal. Do you think it is unnecessary to check again? I checked again because the page was reloaded before the confirmation modal test...